packages/types/src/pricing/common/price-set-money-amount.ts

```diff
@@ -60,6 +60,7 @@ export interface CreatePriceSetMoneyAmountDTO {
   price_set?: PriceSetDTO | string
   price_list?: PriceListDTO | string
   money_amount?: MoneyAmountDTO | string
+  number_rules?: number
```

Comment:
question: by now I am wondering why we need that? also, shouldn't it be rules_count?

Reply:
it's a database field we use to track the number of rules for a given price, that's used when we query for price calculations.

I don't think we should change the database schema in this pr as it's beyond the scope of adding price-rules for prices in price lists. If it's important we can always create a follow-up pr?